dev/source/docs/ros2-sitl.rst

```diff
@@ -12,7 +12,9 @@ Once ROS2 is correctly :ref:`installed <ros2>`, and SITL is also :ref:`installed
     cd ~/ros2_ws/
     colcon build --packages-up-to ardupilot_sitl
     source ~/ros2_ws/install/setup.bash
+    # ArduPilot 4.5
     ros2 launch ardupilot_sitl sitl_dds_udp.launch.py transport:=udp4 refs:=$(ros2 pkg prefix ardupilot_sitl)/share/ardupilot_sitl/config/dds_xrce_profile.xml synthetic_clock:=True wipe:=False model:=quad speedup:=1 slave:=0 instance:=0 defaults:=$(ros2 pkg prefix ardupilot_sitl)/share/ardupilot_sitl/config/default_params/copter.parm,$(ros2 pkg prefix ardupilot_sitl)/share/ardupilot_sitl/config/default_params/dds_udp.parm sim_address:=127.0.0.1 master:=tcp:127.0.0.1:5760 sitl:=127.0.0.1:5501
+    # ArduPilot 4.6 removed the refs argument
```

Comment:
Let me know how to best handle this. 

In ROS, their wiki supports a tab structure like this. can we do that too, or is there another approach.
https://docs.ros.org/en/humble/Tutorials/Beginner-CLI-Tools/Configuring-ROS2-Environment.html#source-the-setup-files

Reply:
We will need to add the spinhx-tabs instructions.
https://sphinx-tabs.readthedocs.io/en/latest/

I'm fighting with the setup scripts to try to install it without dependency conflicts. Can we merge this as-is, and get the sphinx-tabs introduced in a follow-up?